testsuite/simulation/modelica/tearing/Tearing18-minimal.mos

```diff
@@ -28,6 +28,9 @@ val(x8,1.0); getErrorString();
 //     resultFile = "Tearing18_res.mat",
 //     simulationOptions = "startTime = 0.0, stopTime = 1.0, numberOfIntervals = 500, tolerance = 1e-06, method = 'dassl', fileNamePrefix = 'Tearing18', options = '', outputFormat = 'mat', variableFilter = '.*', cflags = '', simflags = ''",
 //     messages = "LOG_SUCCESS       | info    | The initialization finished successfully without homotopy method.
+// assert | debug   | Solving non-linear system 34 failed at time=0.086.
```

Comment:
Why does this test case fail at an assert (only once)?
The loop itself looks correct (nothing teared), so the behaviour should be the same as if no tearing at all was performed. The problem is that the non-linear loop itself is really hard to solve, so sometimes it runs and sometimes it fails with marginal different settings.

My suggestion is to use `status: should fail` (or what ever the equivalent is in out testsuite)

Reply:
Ya. Good catch. I was wondering why as well first. But then I realized the model behaves like this for all tearing methods. 
The test cases `Tearing18-cel.mos`, T`earing18-celMC3.mos`, `Tearing18-minimal.mos `and `Tearing18-omc.mos` behave the same for this non-linear system.

Reply:
Actually on second look, for all the the other tearing methods (`Tearing18-cel.mos`, `Tearing18-celMC3.mos`, and `Tearing18-omc.mos` ) the simulation gets terminated by a assert. For the minimal one it seems to continue and finish properly.

Reply:
Yes, but for `tearingMethod=noTearing` it will fail, too. But the system is exactly the same, so why is it behaving different?

Reply:
I am not sure. As you said the NLS seems to be difficult so solve at that specific time. Any attempt of tearing seems to cause it to fail. Fail as in siumation gets terminated by assert. While the non-teared version (minimal and noTearing) seem to at least be able to continue and finish simulation.